dependency/vault_common_test.go

```diff
@@ -63,4 +63,18 @@ func TestVaultRenewDuration(t *testing.T) {
 	if nonRenewableCertDur < 85 || nonRenewableCertDur > 95 {
 		t.Fatalf("non renewable certificate duration is not within 85%% to 95%%: %f", nonRenewableCertDur)
 	}
+
+	// Secret ID TTL handling
+	data = map[string]interface{}{
+		"secret_id":     "abc",
+		"secret_id_ttl": json.Number("60"),
```

Comment:
We might also want to include a test case for `"secret_id_ttl": json.Number("0"),` and also when that value is entirely absent (e.g. running an older versions of Vault).

Reply:
Updated